realbook/layers/compatibility.py

```diff
@@ -269,9 +270,17 @@ def create_function_from_tensors(
 
     graph_input_names = [t.name for t in graph.inputs]
 
+    graph_def = graph.as_graph_def()
+
+    if not include_control_inputs:
+        # If this graph has any control inputs in it, those inputs will
+        # likely not be convertible (nor do we want them in our converted model!)
+        for node in graph_def.node:
+            node.input[:] = [tensor_name for tensor_name in node.input if not tensor_name.startswith("^")]
```

Comment:
```suggestion
            node.input[:] = [tensor_name for tensor_name in node.input if not importer._IsControlInput(tensor_name)]
```

In order to track future changes in TensorFlow, it's probably better to use `_IsControlInput` directly (even if this function is unlikely to change and the leading `_` indicates you shouldn't use the function directly). You will also need to add `from tensorflow.python.framework import importer` to the top of the file.